vars/submitToTreeherder.groovy

```diff
@@ -42,8 +42,7 @@ def call(String project,
   // TODO include ec2-metadata output in payload
   exchange = "exchange/${PULSE_USR}/jobs"
   routingKey = "${PULSE_USR}.${payload.productName}"
-  schema = libraryResource 'org/mozilla/fxtest/pulse/schemas/treeherder.yml'
-  publishToPulse(exchange, routingKey, JsonOutput.toJson(payload), schema)
+  publishToPulse(exchange, routingKey, JsonOutput.toJson(payload))
```

Comment:
I'd like to raise an issue against this repo, and comment out the code with a reference to the issue. Otherwise I'm concerned that it will be missed. If you like I can take care of this, but it will have to be later today.